packages/components/src/cookies_banner/index.stories.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { text, boolean, withKnobs } from '@storybook/addon-knobs';
+
+import CookiesBanner from './index';
+
+export default { title: 'CookiesBanner', component: CookiesBanner, decorators: [withKnobs] };
+
+export const Default = () => (
+  <BrowserRouter>
```

Comment:
make this `<BrowserRouter>` wrapping a default? @peterhass thoughts?

Reply:
@peterhass pls create a ticket in favro. Need to merge this